pyuvdata/utils.py

```diff
@@ -1184,8 +1195,13 @@ def get_antenna_redundancies(antenna_numbers, antenna_positions, tol=1.0, includ
         List of vectors describing redundant group centers
     lengths : list of float
         List of redundant group baseline lengths in meters
+
+
+    Notes
+    -----
+    Baseline numbers are returned in the convention ant1<ant2.
```

Comment:
I'm confused about this convention. Don't they need to be in particular orientations to be redundant? I see no reason to always return in a specific convention, just stick with the orientations that make them redundant. Then you don't need the `return_conj` option, right?

Reply:
I could have it flip the baseline numbers when `return_conj` is false?

I need the `get_antenna_redundancies` method to be able to return baselines with the `ant1<ant2` convention for `inflate_by_redundancy` to work. `inflate_by_redundancy` first puts the UVData object in `ant1<ant2` form, then uses the conjugate ist from `utils.get_antenna_redundancies` to flip baselines. [Really, this step is done in the UVData.get_antenna_redundancies method]. This ensures that the mapping from compressed -> full doesn't need to conjugate any data along the way.

Reply:
I don't think this is a general solution. It would not work if the baseline you had was one from the group that needed to be oriented the other direction to be redundant. I think the only reason it works in the tests is because you select the first baseline from the group in `compress_by_redundancy`.

Reply:
I'm sorry, but I don't follow... prior to inflating, all the baselines in the object are oriented such that nothing needs to be conjugated or flipped for the redundancy to hold within each group. _This_ is not in the `ant1<ant2` convention. The baseline numbers in the list of groups are also set this way. It's similar to u>0, but using the tolerance.